sbin/wifibox

```diff
@@ -734,9 +739,28 @@ vm_manager() {
 
     log debug "Nice priority: ${_nice_priority}"
     log debug "Arguments: ${_bhyve_args}"
-    ${NICE} -n ${_nice_priority} \
-	    ${BHYVE} ${_bhyve_args} 2>&1 | capture_output debug bhyve
+
+    _kill_child() {
+	local _nice_pid
+	_nice_pid="$(jobs -p)"
+
+	if [ -n "${_nice_pid}" ]; then
+	    log debug "nice_pid=${_nice_pid}"
+	    ${KILL} -SIGTERM ${_nice_pid}
+	else
+	    log info "VM manager: SIGTERM was received but no PID could be found"
+	fi
+    }
+
+    trap _kill_child SIGTERM
+
+    { ${NICE} -n ${_nice_priority} \
+	    ${BHYVE} ${_bhyve_args} 2>&1 | capture_output debug bhyve; } &
+    wait $!
```

Comment:
Before the `wait`, there could be a `log debug` line to take a note that the script is waiting for the VM to stop.